Assets/Animations/Animations/Eminescu-Floating.anim

```diff
@@ -65,7 +65,7 @@ AnimationClip:
         outWeight: 0
       - serializedVersion: 3
         time: 1.6666666
-        value: 0
```

Comment:
Ce ai modificat la animatia asta? Astea par a fi niste boolene, da singurele boolene din animatia asta par neschimbate

Reply:
Aici sincer habar n-am ce s-a întâmplat. Nu am schimbat nimic intenționat.

Reply:
Probabil sunt niște rămășite de când am reparat eu la mine pe branch independent de ce ai făcut tu animațiile. Nu mă prind de ce rezultatul nu a ieșit același totuși.